lib/connectors/crawler/scheduler.rb

```diff
@@ -22,11 +22,50 @@ def connector_settings
         []
       end
 
+      def when_triggered
+        loop do
+          connector_settings.each do |cs|
+            # crawler only supports :sync
+            if sync_triggered?(cs)
+              yield cs, :sync, nil
+              next
+            end
+
+            schedule_key = custom_schedule_triggered(cs)
+            puts('***')
```

Comment:
I guess you will remove these `puts` calls. 

this reminds me of a super nice and helpful [blog post ](https://tenderlovemaking.com/2016/02/05/i-am-a-puts-debuggerer.html)

Reply:
AHH I'll just go bury myself now